Weekly_Work/Week2/Code_Changes_Git.py

```diff
@@ -0,0 +1,159 @@
+import json
+import datetime
+from Commit import Commit
+import pandas as pd
+import matplotlib.pyplot as plt
+import utils
+
+class Code_Changes_Git(Commit):
+    """Class for Code_Changes for Git repositories.
+    
+    Objects are instantiated by specifying a file with the
+    commits obtained by Perceval from a set of repositories.
+        
+    :param path: Path to file with one Perceval JSON document per line
```

Comment:
Again: this would rather be a list of items (obtained by the separate funtion/class reading from the file).

Sorry to insist in this. If you don't see it, let us know, and we follow this way in this iteration, and can retake the issue later on.

Reply:
Oh, I thought this would be the case only for the root class. Will make the required changes now.